webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index',
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        loader: 'babel',
+        exclude: /node_modules/,
+        query: {
+          presets: ['es2015', 'stage-1']
+        }
+      }
+    ]
+  },
+  output: {
+    filename: 'dist/denormalizr.min.js',
+    libraryTarget: 'umd',
+    library: 'denormalizr'
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify('production')
+      }
+    }),
```

Comment:
The comment doesn't answer the question :) I'm curious why do we need `OccurenceOrderPlugin` and the NODE_ENV set to `production` here. `OccurenceOrderPlugin ` ok, no much benefits using it, however the source doesn't include any particular conditions for `production`.